### ISSUE TITLE
Large chat background images are not saved.

### ISSUE THREAD

**Issue by Magnificus36** [telegramdesktop/tdesktop#2866]
### Steps to reproduce
1. In settings change chat background using 'Choose from file' and choose a lage image (20 mega pixel 6.4MB jpg)
2. See that the chosen backgrounds is displayed correctly
3. Quit Telegram
4. Start Telegram
5. The chat background is no longer the image which was chosen in step 1

### Expected behaviour
The chosen chat background should be still set after restarting Telegram. And if the image is much to large then it should be automatically resized and saved.

### Actual behaviour
The chat background changes back to the default image from the gallery, the one with the white flowers, after restarting Telegram. (Smaller images do work.)

### Configuration
**Operating system:** Windows 10 & Windows Server 2008 R2

**Version of Telegram Desktop:** 1.0

**Comment by john-preston** [comment]
@Magnificus36 I could not reproduce that. Is that happening only with a specific image? Can you share it here? Is it happening with small custom backgrounds from file?

**Comment by Magnificus36** [comment]
I did some testing creating images of different sizes and choosing them as chat background. I found that a image of 4433x2956 pixels is not saved and the same image 10% smaller, 3990x2661 pixels, is saved correctly. And apparently it did not matter if it was a nice image or simply a single colour jpg or png.
Originally I had a photo, a jpg of 5372x3648 pixels.

![fails](https://cloud.githubusercontent.com/assets/4574224/22054582/90e040f0-dd56-11e6-9b6c-552290b771c3.png)
![works](https://cloud.githubusercontent.com/assets/4574224/22054581/90e0045a-dd56-11e6-9bd5-86c8bc156469.png)
